requirements-ci.txt

```diff
@@ -1,8 +1,8 @@
-black
-flake8
-pylint
-isort
-fastjsonschema
-pytest
-pytest-asyncio
-pytest-aiohttp
+autopep8==2.1.0
```

Comment:
This file could be moved to `.github/` to reduce clutter in the repository root, since it's now only used in CI.